internal/expr/typeinfo.go

```diff
@@ -9,17 +9,31 @@ import (
 	"sync"
 )
 
+// typeMember should be implemented without pointer recivers as it is used as a
```

Comment:
Minor nitpick, the spelling here for "receivers"

Reply:
Good spot!